index.d.ts

```diff
@@ -18,9 +18,13 @@ export type Options = {
 	readonly basePath?: string;
 
 	/**
-	Remove any paths that match this regexp from the stack.
+	Remove any paths that return false from this callback.
+
+  Example with `path => /unicorn/.test(path)` as `pathFilterCallback`:
+
+	`/Users/sindresorhus/dev/clean-stack/unicorn.js:2:15` → ``
 	*/
-	readonly extraPathRegex?: RegExp;
+	readonly pathFilterCallback?: (string) => boolean;
```

Comment:
```suggestion
	readonly pathFilter?: (string) => boolean;
```

I don't think `Callback` adds anything. The method signature is clear.